vowpalwabbit/lda_core.cc

```diff
@@ -534,7 +532,8 @@ float lda::digamma(float x)
       return ldamath::digamma<float, lda_math_mode::USE_SIMD>(x);
     default:
       // Should not happen.
-      logger::errlog_critical("lda::digamma: Trampled or invalid math mode, aborting");
+      std::cerr << "lda::digamma: Trampled or invalid math mode, aborting" << std::endl;
```

Comment:
there are a couple of extra `;` in this file. Why where these log lines replaced with `std::err`?

Reply:
This is a deep math function and and hitting this `default` indicates a deep issue (we abort here...). Rather than funneling a logger here and potentially having perf impact I figured the message + abort is the closer analogue. Messy situation in general, but I  agree with the abort here